docs/ansible/roles/debops.nfs_server/defaults-detailed.rst

```diff
@@ -112,3 +121,20 @@ networks, with different set of parameters:
 
          - clients: [ '2001:db8:dead:beef::/64', '*.example.org' ]
            options: [ 'rw', 'no_subtree_check', 'no_root_squash' ]
+
+Export the :file:`/usr` directory read-only, by bind-mounting it to the NFS4
+root filesystem, but only after the ZFS service has started.
+Anyone can access it, barring any firewall configuration:
+
+.. code-block:: yaml
+
+   nfs_server__exports:
+     - path: '/srv/nfs/usr'
+       bind:
+        src: '/usr'
+        options:
```

Comment:
The `src` and `options` parameters need to be indented with 2 spaces, not 1. Otherwise, YAML will not be parsed correctly.

Reply:
Did not have any such problem myself, will fix.

Reply:
Done